config/pepperConfig.js.ctmpl

```diff
@@ -247,6 +248,15 @@ var DDP_ENV = {
     {{end}}
 {{end}}
 
+{{if eq $study_guid "rarex"}}
+    {{if eq $environment "dev"}}
+        DDP_ENV['auth0ClientId'] = "gE6vFadpN49ajn76zdSW5FiCqz4aUdlJ";
+        DDP_ENV['auth0Domain'] = "rarex-dev.us.auth0.com";
+        DDP_ENV['auth0Audience'] = "rarex-dev.us.auth0.com";
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+    {{end}}
```

Comment:
Note to self: we'll need to come back here and put stuff in when we move to test/staging/prod.

Reply:
For the sake of completeness, I've added missing configuration blocks.